src/generate/peripheral.rs

```diff
@@ -168,16 +168,24 @@ pub fn render(
     let defaults = p.default_register_properties.derive_from(defaults);
 
     // Push any register or cluster blocks into the output
+    debug!(
+        "Pushing {} register or cluster blocks into output",
+        ercs.len()
+    );
     let mut mod_items = TokenStream::new();
     mod_items.extend(register_or_cluster_block(&ercs, &defaults, None, config)?);
 
+    debug!("Pushing cluster information into output");
```

Comment:
always printing this even when there's no output is a bit strange, but maybe that's a good thing.